presto-resource-group-managers/src/main/java/com/facebook/presto/resourceGroups/db/DatabaseUtil.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.resourceGroups.db;
+
+import org.skife.jdbi.v2.Handle;
+import org.skife.jdbi.v2.IDBI;
+import org.skife.jdbi.v2.exceptions.DBIException;
+
+import java.lang.reflect.InvocationTargetException;
+
+import static com.google.common.reflect.Reflection.newProxy;
+import static java.util.Objects.requireNonNull;
+
+public class DatabaseUtil
```

Comment:
Wait what's going on here - this seems like a totally separate change and should be a different commit